### ISSUE TITLE
Status bar takes up too much of vertical space

### ISSUE THREAD

**Issue by drws** [geany/geany#2513]
More or less, the title already says it all. I think Geany's status bar, implemented in GTK, is too tall. It takes up more than twice the fontsize used in it - or more than 5% of screen real estate on a 720p monitor, if you look at it this way. It would be cool to be able to set this size and spare some much needed pixels.

**Comment by eht16** [comment]
Would you mind sharing a screenshot?
On my system it looks better than you describe: 
![geany_statusbar](https://user-images.githubusercontent.com/617017/82756613-b0008480-9dca-11ea-9e49-52780633a087.png)

The actual appearance might be dependent on the GTK theme and version you are using.

**Comment by codebrainz** [comment]
Sounds like a duplicate of #1419 and #1600.

Mousepad had [a similar bug](https://bugzilla.xfce.org/show_bug.cgi?id=15799) which I investigated and is explained in https://github.com/shimmerproject/Greybird/pull/248.

TLDR; it's a GTK+ bug that can be and is corrected for in some themes, but not others.

**Comment by drws** [comment]
@eht16 Unfortunately I missed your reply before. It looks right on your system. I'm attaching screenshots with the issue which is possibly still relevant.

![image](https://user-images.githubusercontent.com/1191893/129065157-3d9c73cf-a15a-4ef6-84f0-99d330d795da.png)

I'm seeing this issue across variety of systems. Above example is from latest MATE 1.26.0 with GTK3 3.24.30, while this one below is from Ubuntu 18, meaning running under Unity with GTK2. Even this release is newer than the closed issues referenced by @codebrainz, so it should have a solution already incorporated.

![image](https://user-images.githubusercontent.com/1191893/129064752-d1a6cfe1-be9f-4a02-b158-1e0120757ff7.png)

Please take into the account the vertical spacing in the Message Window's tabs and the Status Bar height. In the first example (720p), tabs have comfortable spacing and in the second (900p), it is condensed. While it would be good to have resolution-agnostic middle-ground solution for tabs, both are still acceptable, while the Status Bar height is exaggerated in both cases. I've seen the issue also in XFCE, LXDE, possibly LXQt, with default themes in all these cases. Adwaita is used in the first case and possibly XFCE, but is probably not the reason in every case. Otherwise Geany could notify the users that have it set by default or at least mention the case in the documentation.

**Comment by elextr** [comment]
As @codebrainz noted above, its a theme issue, Geany follows your theme.  The fact that some of them do it "right" shows that.  Having made the decision to follow the theme, its not anything Geany should do.  And all we could really say in the docs is "Some themes look s**t, complain to the theme author, or try another" :smile:.

**Comment by elextr** [comment]
Also I might add, with current Geany being GTK3 you can always override the theme in geany.css.
